util/csvWriter.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
+const fs = require('fs');
+const faker = require('faker');
+const csv = require('fast-csv');
+
+const PHOTO_CHUNK_SIZE = 20;
+
+var generatePhoto = (property_id, photo_id) => {
+	var rounded_photo_id = (PHOTO_CHUNK_SIZE * property_id + photo_id) % 1000;
+	return {
+		photo_id: PHOTO_CHUNK_SIZE * property_id + photo_id, 
+		property_id: property_id,
+		url: `https://s3.amazonaws.com/xillow-photos/${rounded_photo_id}.jpg`,
+	};
+};
+
+var generateDetail = (property_id) => {
+	return {
+		id: property_id,

```

Comment:
indentation is a little off here

Reply:
I think this indentation better allows user to see JSON structure